lib/matplotlib/_constrained_layout.py

```diff
@@ -76,7 +76,7 @@ def do_constrained_layout(fig, h_pad, w_pad,
         Renderer to use.
 
     h_pad, w_pad : float
-      Padding around the axes elements in figure-normalized units.
+      Padding around the axes elements in inches.
```

Comment:
I think this one was correct, because the layout engine does the normalisation before calling this function
https://github.com/matplotlib/matplotlib/blob/be7ea76c49f19ce48958de82a82b3951d9b7f0d2/lib/matplotlib/layout_engine.py#L246-L254

Reply:
....oh, thats why its so confused.  OK, I'll look into it more carefully.